ui/src/mocks/reportWithoutToken.mock.ts

```diff
@@ -396,5 +396,5 @@ export const withoutTokenReport: AppData = {
       "    \"purl\": \"pkg:oci/ubi-minimal@sha256:06d06f15f7b641a78f2512c8817cbecaa1bf549488e273f5ac27ff1654ed33f0?repository_url=registry.access.redhat.com/ubi9/ubi-minimal&tag=9.3-1552&arch=amd64\",\n" +
       "    \"catalogUrl\": \"https://catalog.redhat.com/software/containers/ubi9/ubi-minimal/615bd9b4075b022acc111bf5?architecture=amd64&image=65a828e3cda4984705d45d26\"\n" +
       "  }\n" +
-      "]"
```

Comment:
It seems you're missing here the data similar to what you added to the mixedMock

Reply:
Those 3 fields are optional, so it does not have to be there in all the reports.

Reply:
ok, the comma made me think you wanted to add it.